code/__DEFINES/dcs/signals.dm

```diff
@@ -49,7 +49,7 @@
 // /atom signals
 #define COMSIG_ATOM_CREATED "atom_created"						///from base of atom/proc/Initialize(): sent any time a new atom is created
 #define COMSIG_PARENT_ATTACKBY "atom_attackby"			        ///from base of atom/attackby(): (/obj/item, /mob/living, params)
-	#define COMPONENT_NO_AFTERATTACK 1								//Return this in response if you don't want afterattack to be called
+	#define COMPONENT_NO_AFTERATTACK 1							//Return this in response if you don't want afterattack to be called
```

Comment:
```suggestion
	#define COMPONENT_NO_AFTERATTACK (1<<0)							//Return this in response if you don't want afterattack to be called
```
Flag defines.

Reply:
this entire file needs a pass on that tbqf